site/en/docs/extensions/mv3/promises/index.md

```diff
@@ -71,23 +77,18 @@ should consider using promises in situations such as the following:
 
 * Any time that you want to clean up your code by using a more "synchronous" invocation style.
 * Where error handling would be too difficult using callbacks.
-* When you want a simpler way to invoke a number of concurrent methods and gather the results into a single thread of code.
+* When you want a simpler way to invoke a number of concurrent methods and gather the results into a
```

Comment:
[Avoid "simpler,"](https://developers.google.com/style/word-list#simple) as it may not be simple for all. Instead:

> * When you want to invoke a number of concurrent methods and gather the results into a single thread of code

Reply:
It's not simple, it's simpler than the perviously described solution. It's not the same thing.

Reply:
I had assumed it was related to it being "simpler" for the user, in which case I'd stand by removing it but maybe the suggestion was incorrect and should have suggested a synonym. Condensed? Shorter? 

If these are still off base, feel free to reject/ignore. It's something I noticed while reviewing open PRs, but it's your coverage/call.

Reply:
Revised to "When you want a more condensed way to invoke several concurrent methods and gather the results into a single thread of code."